templates/kubernetes/terraform/environments/dev/application_iam_policy.tf

```diff
@@ -0,0 +1,24 @@
+
+# define policy documents for applications
+# sample policies
+data "aws_iam_policy_document" "resource_access_app1" {
```

Comment:
In all these files can you just make it a single example? The second one doesn't really provide any benefit.